pkgs/os-specific/linux/mbp-modules/mbp2018-bridge-drv/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, kernel, fetchFromGitHub, }:
+
+stdenv.mkDerivation rec {
+  name = "mbp2018-bridge-drv";
+  version = "b43fcc069da73e051072fde24af4014c9c487286";
+
+  src = fetchFromGitHub {
+    owner = "MCMrARM";
+    repo = "mbp2018-bridge-drv";
+    rev = "0.01";
+    sha256 = "0ac2l51ybfrvg8m36x67rsvgjqs1vwp7c89ssvbjkrcq3y4qdb53";
+  };
+
+  buildPhase = ''
+    make -C ${kernel.dev}/lib/modules/${kernel.modDirVersion}/build \
+      M=$(pwd) modules
```

Comment:
I think ``-j$NIX_BUILD_CORES`` is missing.

Reply:
wont hurt to add it, but maybe it's already declared with the kernel module wrapper, not sure. Either way, adding it...

Reply:
done